jobs/config.json

```diff
@@ -7793,7 +7793,8 @@
       "--env=KOPS_PUBLISH_GREEN_PATH=gs://kops-ci/bin/latest-ci-gce-green.txt",
       "--env-file=jobs/platform/kops_gce.env",
       "--extract=ci/latest",
-      "--ginkgo-parallel",
+      "--ginkgo-parallel=30",
```

Comment:
I think I just made a mistake before, though I _think_ it worked before.  I suspect I was silently dropping a flag.